core/templates/dev/head/css/oppia.css

```diff
@@ -3921,17 +3921,26 @@ md-card.preview-conversation-skin-supplemental-card {
   font-size: 1em;
   font-weight: normal;
   line-height: 1.2em;
-  margin: 0.3em 0.5em;
+  margin: 0.3em 0.5em;  
   padding: 0.2em;
   position: absolute;
   word-break: break-all;
 }
-
 .oppia-activity-summary-tile .objective {
-  height: 60px;
+  display: -webkit-box;
   font-family: "Rubik", "Roboto", Arial, sans-serif;
   font-size: 0.85em;
+  height: 70px;
+  line-height: 1.26;
+  /* `max-height` is calculated as: 1.26 (line-height) * 3 (lines) / 0.85
+  (font-size). An additional (1.26 (line-height) * 0.85 (font-size) * 0.25) is added
+  to account for letters (like y,j and g) that fall beneath the baseline. */
```

Comment:
Hi @rohan-gulati, one request: please could you reply to all comments, following the instructions in step 5 [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)? In particular, for comments that you've addressed, please reply "Done." -- this helps keep track of which comments have been addressed and which haven't. Thanks!

Also, just wanted to flag the last question in my previous comment. If you have the max-height, is -webkit-line-clamp redundant?